pkg/blockdevice/new_block_device_from_file_windows.go

```diff
@@ -0,0 +1,95 @@
+//go:build windows
+// +build windows
+
+package blockdevice
+
+import (
+	"syscall"
+	"unsafe"
+
+	"github.com/buildbarn/bb-storage/pkg/util"
+
+	"golang.org/x/sys/windows"
+)
+
+var (
+	kernel32          = windows.NewLazySystemDLL("kernel32.dll")
+	GetDiskFreeSpaceW = kernel32.NewProc("GetDiskFreeSpaceW")
+)
```

Comment:
I know that we lazily load the DLL, but does calling NewProc() against it still make it lazy? If not, what are your thoughts on putting the initialization of these fields inside a `sync.Once` in `NewBlockDeviceFromFile()`? That way we only depend on this DLL if block devices are used.

Or are we convinced that `kernel32.dll` is so fundamental, that an unconditional dependency doesn't harm?

Reply:
Oops sorry.  Done!